secondPush

```diff
@@ -0,0 +1 @@
+This is my second push. The first push was not created in another branch. This one is created in another branch and I will next learn how to merge this file into the other branch with Github!
```

Comment:
This is my first line note, this section is meant for general comments, opposed to line comments are intended for specific code evaluation.